e2e/tests/steps_init_test.go

```diff
@@ -34,8 +34,8 @@ var (
 
 const (
 	// defines the host & port scheme that LocalNet uses for naming validators.
-	// e.g. v1-validator-001 thru v1-validator-999
-	validatorServiceURLTmpl = "v1-validator%s:%d"
+	// e.g. validator-001 thru validator-999
+	validatorServiceURLTmpl = "validator-%s-pocket-validator:%d"
```

Comment:
@dylanlott the test appears to be green in Tilt with that change. I think we're good! :) Thanks for bringing that to my attention.

Reply:
Amazing! 💯 Glad it worked as expected!